architecture/mce-overview-ocp.adoc

```diff
@@ -1,12 +1,12 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="mce-overview-ocp"]
 include::_attributes/common-attributes.adoc[]
-= About multicluster engine for Kubernetes operator
+= About {mce}
```

Comment:
No capitalization, so not a proper name.

But no article either, which does not look right. I would suggest "the {mce}" here, especially since "the {mce}" and "the {mce-short}" is already used elsewhere in this PR.